testmodels/src/main/java/com/amplifyframework/testmodels/commentsblog/Comment.java

```diff
@@ -20,9 +20,9 @@
 @SuppressWarnings("all")
 @ModelConfig(pluralName = "Comments")
 public final class Comment implements Model {
-  public static final QueryField ID = field("id");
-  public static final QueryField CONTENT = field("content");
-  public static final QueryField POST = field("commentPostId");
+  public static final QueryField ID = field("Comment", "id");
```

Comment:
Do you still have some test coverage older the old, `null` modelName code path, now?

Reply:
I only updated this particular package, but I agree that it's ideal to test that path as well. Will fix.

Reply:
Thanks!